### PR TITLE
minor fixes in the sampling RFC

### PR DESCRIPTION
**Pull request by jahtalab**

### CODE REVIEW

**Comment by jahtalab** [comment]
I signed it

File: text/0006-sampling.md [hunk 1]
```diff
@@ -174,11 +174,11 @@ instrumenting code and never access `SampledFlag` unless used in context propaga
 This is a new concept added in the OpenTelemetry API that allows to suggest sampling hints to the
 implementation of the API:
  * `NOT_RECORD`
-   * Suggest to not `RecordEvents = false` and not propagate `SampledFlag = false`.
+   * Suggest to not record (`RecordEvents = false`) and not propagate (`SampledFlag = false`).
```
Comment (songy23) [inline text/0006-sampling.md]:
```suggestion
   * Suggest to not set (`RecordEvents = false`) and not propagate (`SampledFlag = false`).
```
Same for the two lines below.

**Comment by bogdandrutu** [comment]
@jahtalab please address feedback :)

**Comment by jahtalab** [comment]
@bogdandrutu , please have another look.

**Comment by jahtalab** [comment]
Are there any changes required for this PR? 

cc @songy23, @tedsuo

**Comment by bogdandrutu** [comment]
Merging this as admin, only cleanups.

**Pull request merged**